pageserver/src/layered_repository.rs

```diff
@@ -662,15 +665,15 @@ impl Repository {
         Ok(tenant_conf)
     }
 
-    pub fn persist_tenant_config(
-        conf: &'static PageServerConf,
-        tenant_id: ZTenantId,
+    pub fn crashsafe_persist_tenant_config(
```

Comment:
Nit: IMO persist implies that it is crashsafe, e g save_metadata is just save_metadata without crashsafe

Reply:
Removed the `crashsafe_` prefix.